### ISSUE TITLE
unimportant feature request: additional vid discount and runic shield slot checks

### ISSUE THREAD

**Issue by TheOldOne822** [unicornbloods/Thaumic-Mixins#8]
Thaumcraft is coded to only check the 4 vanilla armor slots and the 4 baubles added slots from the normal version of baubles for calculating runic shield level and only those plus the wand being used for vis discount. I was wondering if it would be possible to change it so it also checks slots added by other mods. 
Examples:
additional up to 16 baubles slots (total up to 20) if using baubles expanded (It will only check the first 4 baubles expanded slots)
4 slots added by Travelers gear
7 slots added by Tinkers Construct (the 3 heart container slots can likely be ignored)
3? slots added by Aquaculture (Last used it years ago so don't remember the number)

There may be more mods adding slots but those are the ones I know off the top of my
Aquaculture can likely also be ignored as if i recall correctly its slots are "ring" and "amulet" so if someone is making a Thaumcraft addon that adds one of those they would use the baubles version of the slot

I don't know if it would be changeable without a coremod modifying that part of the Thaumcraft code so expect it to be beyond your plans.

**Comment by unicornbloods** [comment]
It doesn't seem that hard. I will look into it. The vis discount probably isn't so useful though, since you can easily hit the cap with existing items in Thaumcraft add-ons.

**Comment by TheOldOne822** [comment]
Yes it isn't hard to max out the vis discount, that's more about not having to remember to keep any vis discount ring in the first 4 bauble slots early game and the thought I once had of adding a vis discount modifier to the Tinkers Construct gloves just so there was another option on them other than increasing mining speed

**Comment by unicornbloods** [comment]
That is fair. I would love to see more modifiers for Tinker's Construct to pair with Thaumcraft. If you are in GTNH or Legacy Modding discord send me a ping to discuss this particular thing more.

As for supporting the slots, the main hurdle is the time I have available between work and preparing for winter, I will get to it this week, though. Plus I am finishing up a different feature.
